connect/client.go

```diff
@@ -106,6 +109,53 @@ type restClient struct {
 	client    httpClient
 }
 
+func (rs *restClient) GetFiles(itemUUID string, vaultUUID string) ([]onepassword.File, error) {
+	span := rs.tracer.StartSpan("GetFiles")
+	defer span.Finish()
+
+	jsonURL := fmt.Sprintf("/v1/vaults/%s/items/%s/files", vaultUUID, itemUUID)
+	request, err := rs.buildRequest(http.MethodGet, jsonURL, http.NoBody, span)
+	if err != nil {
+		return nil, err
+	}
+	response, err := rs.client.Do(request)
+	if err != nil {
+		return nil, err
+	}
+	if err := expectMinimumConnectVersion(response, version{1, 3, 0}); err != nil {
+		return nil, err
+	}
+	var files []onepassword.File
+	if err := parseResponse(response, http.StatusOK, &files); err != nil {
+		return nil, err
+	}
+
+	return files, nil
+}
+
+func (rs *restClient) DownloadFile(fileUUID, itemUUID, vaultUUID, targetDirectory string) error {
```

Comment:
Do we have a specific reason for having the `fileUUID, itemUUID, vaultUUID` in the signature instead of `*onepassword.File`? I am not sure yet which one of those would be the better option. So I am happy to be convinced of one of them.

Reply:
From what I see, `*onepassword.File` does not store any information whatsoever about the `itemUUID` or the `vaultUUID`, so we'd have to pass these parameters as well, in which case we'd only replace the `fileUUID`. This seems a bit of a hassle, so I would much rather keep the function signature the way it currently is. Let me know if I am missing anything here.

Reply:
What do we need the `vaultUUID` and `itemUUID` for if we already have the `*onepassword.File`?

Reply:
I overlooked the fact that the vaultUUID and itemUUID are, in fact, already within `*onepassword.File`, under the `ContentPath` parameter. I adjusted it accordingly, it is indeed way cleaner this way.